deployment/ccip/test_helpers.go

```diff
@@ -268,6 +274,28 @@ func CCIPSendRequest(
 	return tx, blockNum, nil
 }
 
+// CCIPSendCalldata packs the calldata for the Router's ccipSend method.
+// This is expected to be used in Multicall scenarios (i.e multiple ccipSend calls
+// in a single transaction).
+// Multicalls can't be made with fee token 0x0 due to the Multicall itself not
```

Comment:
Are you not sending the value as fee here in `genMessages` instead of using feetoken - https://github.com/smartcontractkit/chainlink/pull/15292/files#diff-a0967558956b7b4a235492bf27d9cb41b7b7a2e0d05b9cab4f9e2871b0893942R294 ?

Reply:
Oops yeah this is a stale comment, good catch